main.css

```diff
@@ -0,0 +1,100 @@
+* {
+    box-sizing: border-box;
+  }
+  
+body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Urbanist', sans-serif;
+    font-size: 18px;
+}
+
+.main-container-Santouka {
+    width: 80%;
+    height: 250px;
+    margin: 20px auto 0px 0px;
+    background: black;
+    display: flex;
+    justify-content: space-around;
+}
+
+.santouka-img {
+    display: block;
+}
+
+.santouka-img img {
+    padding: 20px;
+}
+
+.santouka-desc {
+    display: block;
+}
+
+.santouka-desc h1 {
+    margin: 0;
+    padding: 20px;
+    color:white;
+    letter-spacing: 0.12em;
+}
+
+.santouka-desc p {
+    color: rgb(97, 97, 235);
+    display: inline;
+}
+
+.santouka-desc h2 {
+    font-size: 14px;
+    color: white;
+    font-weight: normal;
+    line-height: 1.5;
+    margin: 0;
+    padding: 10px 20px;
+}
+
+.main-container-Gojiro {
```

Comment:
same. case insensiteive, so lets use lower case only!

JS is case sensitive, so its ok in JS.
CSS, not!